frontend/src/Pages/Student/Course/EnrolledCourses.jsx

```
@@ -0,0 +1,169 @@
+                                    <p className="text-lg font-semibold mt-2">
+                                        Class Assessment Marks: <span className="text-yellow-300">{enrollment.class_assessment_marks}</span>
+                                    </p>
+                                </div>
+                            </div>
+
+                            {/* Re-Enroll Button */}
+                            {enrollment.canReEnroll && (
+                                <button
+                                    onClick={() => handleReEnroll(enrollment.course_session.course.id)}
+                                    className="mt-4 sm:mt-0 bg-blue-500 hover:bg-blue-600 text-white font-bold py-2 px-4 rounded-lg shadow-md"
+                                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)